src/python/antares_xpansion/sensitivity_driver.py

```diff
@@ -36,8 +36,9 @@ def launch(
         self.simulation_output_path = self._get_simulation_output_path(
             simulation_output_path
         )
-        with zipfile.ZipFile(self.simulation_output_path, 'r') as output_zip:
-            output_zip.extractall(xpansion_simulation_output)
+        # if (os.path.splitext(simulation_output_path)[1] == ".zip"):
+        #     with zipfile.ZipFile(self.simulation_output_path, 'r') as output_zip:
+        #         output_zip.extractall(xpansion_simulation_output)
```

Comment:
Remove this code ?

Reply:
done before your comment but it was not visbible in github!